Sources/Client/Views/Play/PlayServerView.swift

```diff
@@ -25,6 +25,8 @@ struct PlayServerView: View {
   var serverDescriptor: ServerDescriptor
   var renderCoordinator: RenderCoordinatorProtocol
   
+
+  // Link whether the cursor is captured to whether input gets sent to delegate
```

Comment:
How did this comment get there? I think it was just explaining the behaviour of one of the lines in the init

Reply:
Probably same reason as this https://github.com/stackotter/delta-client/pull/41#discussion_r744137062, will change